src/storage/statistics_sr.c

```diff
@@ -241,16 +241,8 @@ xstats_update_statistics (THREAD_ENTRY * thread_p, OID * class_id_p, bool with_f
   assert (npages > 0);
   cls_info_p->ci_tot_pages = MAX (npages, 0);
 
-  estimated_nobjs = heap_estimate_num_objects (thread_p, &(cls_info_p->ci_hfid));
-  if (estimated_nobjs == -1)
-    {
-      /* cannot get estimates from the heap, use old info */
-      assert (cls_info_p->ci_tot_objects >= 0);
-    }
-  else
-    {
-      cls_info_p->ci_tot_objects = estimated_nobjs;
-    }
+  /* use value from "select --+ sampling count(*) ..." */
```

Comment:
It seems that `estimated_nobjs` is no longer necessary.

Reply:
Okay. I'll fix it later.